apiserver/facades/client/machinemanager/machinemanager.go

```diff
@@ -112,6 +114,13 @@ type MachineService interface {
 	HardwareCharacteristics(ctx context.Context, machineUUID string) (*instance.HardwareCharacteristics, error)
 }
 
+type ApplicationService interface {
```

Comment:
Comment me.

Reply:
One day I will submit a PR where I haven't forgotten to comment an interface!